WireDomain/Sources/WireDomain/Repositories/Conversations/ConversationRepository.swift

```diff
@@ -194,6 +195,7 @@ public final class ConversationRepository: ConversationRepositoryProtocol {
     private let conversationsLocalStore: any ConversationLocalStoreProtocol
     private let userRepository: any UserRepositoryProtocol
     private let teamRepository: any TeamRepositoryProtocol
+    private let userClientsRepository: any UserClientsRepositoryProtocol
```

Comment:
Is this being used?

Reply:
no, removed in https://github.com/wireapp/wire-ios/pull/2164/commits/6b004359cdfeb2215764967ca051b578a1238a7a